src/app/screen-snippet-handler.ts

```diff
@@ -107,14 +106,15 @@ class ScreenSnippet {
     try {
       await this.execCmd(this.captureUtil, this.captureUtilArgs);
       if (windowHandler.isMana) {
-        const dimensions = this.getImageSize();
-        const imageSize = { width: dimensions?.width || -1, height: dimensions?.height || -1 };
-        if (imageSize.width === -1 || imageSize.width === -1) {
+        logger.info('screen-snippet-handler: Attempting to extract image dimensions from: ' + this.outputFilePath);
```

Comment:
```suggestion
        logger.info(`screen-snippet-handler: Attempting to extract image dimensions from: ${this.outputFilePath}`);
```

Reply:
I honestly prefer the + over the ${}, unless it is a lot of mixed strings and objects. I think it is easier to read. At one time, you reach the point where many + gets crowded and than  ${} becomes more readable, but when it is like this, just a plus object in the end, I prefer the plus.

Reply:
Yeah, I don't have any issue there, it's a matter of style. Your choice ultimately.